src/MenuBar.tsx

```diff
@@ -6,7 +6,7 @@ import {
   type MenuBarItem as _MenuBarItem,
   type SubMenuItem as _SubMenuItem,
 } from './generated/MenuBar.js';
-import { mapItemsWithComponents } from './utils/mapItemsWithComponents.js';
+import { getOriginalItem, mapItemsWithComponents } from './utils/mapItemsWithComponents.js';
 
 export * from './generated/MenuBar.js';
```

Comment:
Looks like the exported `MenuBarItemSelectedEvent` also needs to be updated. Working on it.

Reply:
Done